scripts/update/Updater.php

```diff
@@ -1322,5 +1329,46 @@ public function update($initialVersion) {
         }
 
         $this->skip('10.1.0', '10.3.0');
+
+        if ($this->isVersion('10.3.0')) {
+
+            // Install the synchronisation service
+            if ($this->getServiceManager()->has(SynchronisationService::SERVICE_ID)) {
```

Comment:
Why do you duplicate the behavior of the install scripts instead of directly call them? It should be the same, and should also simplify the code